doc/making-a-visualizer.md

```diff
@@ -270,9 +315,7 @@ the file containing its class definition and export module in the folder
 `src/visualizers`. When the frontscope client runs, it'll find your visualizer
 and compile it at runtime.
 
-The visualizers in `src/visualizers` will appear as options when you start
-Numberscope with `npm run dev` as usual.
-
+If running `npm run dev`, visualizers in `src/visualizers` will be available.
 As discussed [earlier](#develop-your-visualizer-on-the-workbench), visualizers
 that aren't ready for Numberscope users should go in
```

Comment:
Do you want to add a remark highlighting the fact that in particular, the P5VisualizerTemplate visualizer that's been discussed throughout this document will only be visible if you `npm run dev:workbench`?

Reply:
got it